src/main/resources/db/migration/all/20230123154608__add_probation_region_to_users.sql

```diff
@@ -1,2 +1,8 @@
-ALTER TABLE "users" ADD COLUMN probation_region_id UUID NOT NULL;
+ALTER TABLE "users" ADD COLUMN probation_region_id UUID;
 ALTER TABLE "users" ADD FOREIGN KEY (probation_region_id) REFERENCES probation_regions(id);
+
+-- Give existing users a default region.
+UPDATE "users"
+SET probation_region_id = 'a02b7727-63aa-46f2-80f1-e0b05b31903c'; -- North West
```

Comment:
Could we make this more rigorous with a subquery, rather than a UUID, i.e:

```suggestion
SET probation_region_id = (
        SELECT
          id
        FROM
          probation_regions
        WHERE
          delius_code = 'N51'
        LIMIT 1
      );
```